src/lib/Validator/Constraints/EmailInvitation.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\User\Validator\Constraints;
+
+use JMS\TranslationBundle\Model\Message;
+use JMS\TranslationBundle\Translation\TranslationContainerInterface;
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class EmailInvitation extends Constraint implements TranslationContainerInterface
+{
+    public string $message = 'ibexa.user.invitation.user_with_email_exists';
+
+    public static function getTranslationMessages()
+    {
+        return [
+            Message::create('ibexa.user.invitation.user_with_email_exists', 'validators')
```

Comment:
```suggestion
            Message::create($message, 'validators')
```

Reply:
That's not gonna work, I would have to create property & private const, and that's just seems to much.